client/js/README.md

```diff
@@ -21,9 +21,9 @@ The client function connects to the API of a hosted Gradio space and returns an
 The simplest example looks like this:
 
 ```ts
-import { client } from "@gradio/client";
+import { Client } from "@gradio/client";
```

Comment:
We should update the section header to be `Client` instead of `client`, and in the paragraph above, replace "client function" with e.g. `Client.client` factory function